config/initializers/ahoy.rb

```diff
@@ -6,7 +6,7 @@
 module Ahoy
   class Store < Ahoy::Stores::LogStore
     def exclude?
-      return if FeatureManagement.enable_load_testing_mode?
+      return true if FeatureManagement.enable_load_testing_mode?
```

Comment:
I take back that last part about changing the setting on production. That's not a good idea. I would first try commenting out the entire `exclude?` method on `int` or `qa` and see if that changes anything.